### ISSUE TITLE
Training loss Nan

### ISSUE THREAD

**Issue by EdenGabriel** [NVlabs/RVT#87]
Sorry to disturb you, but i indeed need your help. 

When I train rvt2, I get loss nan after 2 epochs, and I use config in model_99 ([pretrained RVT-2 model](https://huggingface.co/ankgoyal/rvt/tree/main/rvt2))

**Comment by EdenGabriel** [comment]
I have solved it. 

I changed the Python version from 3.12 to 3.10 and downgraded the PyTorch version, which resolved the issue.
